gap/FinSetsForCAP.gi

```diff
@@ -770,6 +781,42 @@ AddLift( category_of_finite_sets,
     
 end );
 
+## beta \circ alpha^{-1} is again an ordinary function,
+## i.e., fibers of alpha are mapped under beta to the same element
+AddIsColiftable( category_of_finite_sets,
+  function ( category_of_finite_sets, alpha, beta )
+    
+    return ForAll( AsList( ImageObject( alpha ) ),
+                   i -> Length( DuplicateFreeList( List( Preimage( alpha, FinSetNC( category_of_finite_sets, [ i ] ) ), beta ) ) ) = 1 );
```

Comment:
We must not use `DuplicateFreeList` because that would compare elements via `=`, but we compare elements via `IsEqualForElementsOfFinSets`.